tests/spec/time-flags.spec.js

```diff
@@ -104,3 +104,45 @@ describe('node times (atime, mtime, ctime) with mount flags', function() {
     });
   });
 });
+
+function memoryFS(flags, callback) {
```

Comment:
When you're working on existing code in an open source project, it's important not to duplicate code.  Every time we have the same code more than once, it makes the project harder to maintain, since a bug in the code has to be fixed multiple times.  You'll hear devs say to DRY your code out, meaning "Don't Repeat Yourself."  In this case, you've copied the `memoryFS` function here, since it wasn't in the scope you wanted.  A better solution would have been to move your test into the proper scope so you could just call it.  Or, you might have refactored the method so it was available in more than one scope.